test/common.jl

```diff
@@ -59,3 +59,126 @@ function get_branch(sys::System, other::Branch)
 
     error("Did not find branch with buses $(other.arc.from.name) ", "$(other.arc.to.name)")
 end
+
+function create_system_with_dynamic_inverter()
```

Comment:
@jd-lara  I moved your code so that I wouldn't have to duplicate all of it.  If you prefer that I didn't do this, I can revert it.

Reply:
This is fine